config/import-control.xml

```diff
@@ -39,8 +39,6 @@
   <allow class="java.lang.annotation.RetentionPolicy" local-only="true"/>
   <allow class="java.lang.annotation.Target" local-only="true"/>
   <allow class="com.puppycrawl.tools.checkstyle.xpath.XpathQueryGenerator" local-only="true"/>
-  <allow class="com.puppycrawl.tools.checkstyle.FileStatefulCheck"/>
-  <allow class="com.puppycrawl.tools.checkstyle.StatelessCheck"/>
```

Comment:
Clone service will be in this package, so the imports need to stay.
See https://github.com/checkstyle/checkstyle/pull/4882/files#diff-404a4bfa3acb4cbc7635f4b2f27435b1

Reply:
we will move them back when it become required.